README.md

```diff
@@ -2,6 +2,220 @@
 
 [![Build Status](https://travis-ci.org/revdotcom/revai-java-sdk.svg?branch=develop)](https://travis-ci.org/revdotcom/revai-java-sdk)
 
+## Documentation
+
+See the [API docs](https://www.ai.rev.ai/docs) for more information about the API.
+
+## Install the SDK
+The recommended way to use the Rev.ai Java SDK is to import it into the project using Maven.
+
+      <dependency>
+        <groupId>ai.rev</groupId>
```

Comment:
The reverse domain format is a bit confusing with short domain names and non-dotcom domain suffixes.

As I understand it, this is a requirement for our groupId to match our domain name. Is this correct?

Reply:
Yes and this follows maven groupid naming convention.
http://maven.apache.org/guides/mini/guide-naming-conventions.html